client/my-sites/sidebar/sidebar.jsx

```diff
@@ -345,7 +344,9 @@ export class MySitesSidebar extends Component {
 		return (
 			<li className={ this.itemLinkClass( [ '/plans' ], linkClass ) }>
 				<a onClick={ this.trackUpgradeClick } href={ planLink }>
-					<JetpackLogo size={ 24 } />
```

Comment:
I think we should add a condition to the JetpackLogo and return a different svg instead of doing this.

Reply:
@enejb How about something like this? https://github.com/Automattic/social-logos/pull/55

Reply:
I think its good to have just one component that returns what is expected. This way we could show different icons based on what size is require but use the same component.

Reply:
Also next time we change it there will only be one place from to do this. 
It makes also sense to me if we have 2 different version of the logo displayed in https://wpcalypso.wordpress.com/devdocs/design/jetpack-logo 24 and 32 size. 
and https://wpcalypso.wordpress.com/devdocs/design/jetpack-colophon

Reply:
@enejb Hmm. That's a good point. I'm concerned with context, though. The `social-icons` set is designed like Gridicons to be used on the 24px scale. The "32px" icon version isn't pixel-hinted for 32px so it can be at any size (albeit blurry).